src/EyesVisualGrid.js

```diff
@@ -55,16 +55,20 @@ class EyesVisualGrid extends EyesBase {
   }
 
   /**
-   * @signature `open(driver, configuration)`
-   * @signature `open(driver, appName, testName, ?viewportSize, ?configuration)`
+   * @signature `open(driver)`
+   * @sigparam {object} driver - The web driver that controls the browser hosting the application under test.
+   * @signature `open(driver, appName, testName, viewportSize)`
+   * @sigparam {string} appName - The name of the application under the test.
+   * @sigparam {string} testName - The test name.
+   * @sigparam {RectangleSize|RectangleSizeObject} [viewportSize] - The required browser's viewport size
    *
-   * @param {object} driver The web driver that controls the browser hosting the application under test.
-   * @param {Configuration|string} optArg1 The Configuration for the test or the name of the application under the test.
-   * @param {string} [optArg2] The test name.
-   * @param {RectangleSize|RectangleSizeObject} [optArg3] The required browser's viewport size
+   * @param {object} driver - The web driver that controls the browser hosting the application under test.
+   * @param {Configuration|string} optArg1 - The Configuration for the test or the name of the application under the test.
```

Comment:
we don't pass Configuration to open anymore. Should be just
   * @param {String} [appName] - Application name
Check last version in develop branch